unityPackage/Runtime/Scripts/Rig_Bridge/test/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+project( rig2csTest CSharp)

```

Comment:
I recognize this as the rig2csTest already in the repository. Is this something different?

Reply:
I removed the whole folder and updated the ReadMe to include steps to add your rig2cs folder to the users Unity Project.